src/Controllers/SearchController.cs

```diff
@@ -105,10 +153,17 @@ public async Task<IEnumerable<string>> GetSearchSuggestions(string query)
             .Distinct()
             .ToList();
 
+        if (schemaLanguages != null && schemaLanguages.All(s => string.IsNullOrEmpty(s))) schemaLanguages = null;
+        if (dependsOnModels != null && dependsOnModels.All(d => string.IsNullOrEmpty(d))) dependsOnModels = null;
+        if (issuers != null && issuers.All(i => string.IsNullOrEmpty(i))) issuers = null;
+
         return context.Repositories
             .Include(r => r.SubsidiarySites)
             .Include(r => r.ParentSites)
             .Include(r => r.Models
+                .Where(m => schemaLanguages == null || schemaLanguages.Any(s => m.SchemaLanguage == s))
+                .Where(m => dependsOnModels == null || dependsOnModels.Any(d => m.DependsOnModel.Contains(d)))
```

Comment:
Q: Sollte dieser Filter nicht unten bei Zeile 174 beim Filtern der `query` berücksichtigt werden?

Reply:
Ich vestehe nicht ganz was du meinst.
Mein Verständnis: Im Where Zeile 164 filtere ich die Modelle in denen gesucht werden soll, anhand des Filterparameters. Auf Zeile 174 wird dann in den verbleibenden Modellen mit dem Suchstring gesucht.

Reply:
In diesem Fall stimmt das. Ich hatte die Annahme, dass der Suchstring und der Filter die gleiche Gewichtung haben. Das passt also so 👍🏻